mpas_analysis/test/test_climatology.py

```diff
@@ -110,13 +110,17 @@ def test_get_mpas_climatology_file_names(self):
         config = self.setup_config()
         fieldName = 'sst'
         monthNames = 'JFM'
-        (climatologyFileName, regriddedFileName) = \
+        (climatologyFileName, climatologyPrefix, regriddedFileName) = \
             climatology.get_mpas_climatology_file_names(config, fieldName,
                                                         monthNames)
         expectedClimatologyFileName = '{}/clim/mpas/sst_QU240_JFM_' \
                                       'years0002-0002.nc'.format(self.test_dir)
         self.assertEqual(climatologyFileName, expectedClimatologyFileName)
 
+        expectedClimatologyPrefix = '{}/clim/mpas/sst_QU240_' \
+                                    'JFM'.format(self.test_dir)
+        self.assertEqual(climatologyPrefix, expectedClimatologyPrefix)
+
```

Comment:
I don't like unit tests for unit testings' sake but I do like not having bugs later.  If you have a hint this complexity will produce problems later please add unit tests, otherwise, I wouldn't worry about it.